pkg/agent/agent.go

```diff
@@ -146,6 +165,24 @@ func getRelayAgentConfig(ctx context.Context) error {
 	return nil
 }
 
+func getRelayAgentNamespace(ctx context.Context) error {
```

Comment:
This func is setting fingerprint value although it is named as `getRealyAgentNamespace`. Maybe rename it to `setAgentFingerprint`?

Reply:
renamed it to setAgentFingerprint